tests/integration/tests/autocomplete.js

```diff
@@ -1,303 +1,306 @@
-import {clearStore, initBrowser, wait} from '../tools'
-import AutocompleteHelper from "../helpers/autocomplete";
-import ResponseHandler from "../helpers/response_handler";
-import {autocomplete} from '../../../config/constants.yml';
-const configBuilder = require('@qwant/nconf-builder')
-const config = configBuilder.get()
-const APP_URL = `http://localhost:${config.PORT}`
-
-const SUGGEST_MAX_ITEMS = config.services.geocoder.max_items
-
-let browser
-let page
-let autocompleteHelper
-let responseHandler
-const mockAutocomplete = require('../../__data__/autocomplete.json')
-const mockAutocompleteAllTypes = require('../../__data__/autocomplete_type.json')
-
-beforeAll(async () => {
-  let browserPage = await initBrowser()
-  page = browserPage.page
-  browser = browserPage.browser
-  responseHandler = await ResponseHandler.init(page)
-  autocompleteHelper = new AutocompleteHelper(page)
-})
-
-test('search and clear', async () => {
-  expect.assertions(4)
-  responseHandler.addPreparedResponse(mockAutocomplete, /autocomplete\?q=Hello/)
-  responseHandler.addPreparedResponse(mockAutocomplete, /autocomplete\?q=Helloa/)
-  await page.goto(APP_URL)
-  await autocompleteHelper.typeAndWait('Hello')
-  let cleanHandle = await autocompleteHelper.getClearFieldButton()
-  expect(cleanHandle).not.toBeNull()
-
-  const autocompleteItems = await autocompleteHelper.getSuggestList()
-  expect(autocompleteItems.length).toEqual(SUGGEST_MAX_ITEMS)
-
-
-  let searchValue = await autocompleteHelper.getSearchInputValue()
-  expect(searchValue).toEqual('Hello')
-
-  await autocompleteHelper.clearField()
-  let searchValueAfterClear = await autocompleteHelper.getSearchInputValue()
-  expect(searchValueAfterClear).toEqual('')
-})
-
-test('search has lang in query', async () => {
-  const langPage = await browser.newPage()
-  await langPage.setDefaultTimeout(2000) // to raise Puppeteer timeout early on fail
-  await langPage.setExtraHTTPHeaders({
-    'accept-language': 'de,en;q=0.8'
-  })
-  const responseHandler = await ResponseHandler.init(langPage)
-  const autocompleteHelper = new AutocompleteHelper(langPage)
+/* globals Poi */
+import {clearStore, initBrowser, wait} from '../tools';
+import AutocompleteHelper from '../helpers/autocomplete';
+import ResponseHandler from '../helpers/response_handler';
+const configBuilder = require('@qwant/nconf-builder');
+const config = configBuilder.get();
+const APP_URL = `http://localhost:${config.PORT}`;
+
+const SUGGEST_MAX_ITEMS = config.services.geocoder.max_items;
+
+let browser;
+let page;
+let autocompleteHelper;
+let responseHandler;
+const mockAutocomplete = require('../../__data__/autocomplete.json');
+const mockAutocompleteAllTypes = require('../../__data__/autocomplete_type.json');
+
+beforeAll(async() => {
```

Comment:
No more whitespace after `async`?

Reply:
This is a result of `eslint --fix`, and it doesn't seem to be invalid code. But I agree this is unclear. I'll change.